### PR TITLE
248-update-content-on-duns-number-page

### PR DESCRIPTION
**Pull request by benvand**
https://trello.com/c/6vSysI7W/248-update-content-on-duns-number-page

* Update create_duns_number.html with content suggestions
  * https://docs.google.com/presentation/d/1sukhMNNbGTl3KDYYVedtBcNT5gQliaXjbrGW5Mrf2ak/edit#slide=id.g759a4b2e43_0_2
  * Discussion with Nora
* Update error messages on supplier registration forms as per:
  * Content guidlines here:
    * https://design-system.service.gov.uk/components/error-message/
  * Discussion with Nora:
    * https://docs.google.com/presentation/d/1sukhMNNbGTl3KDYYVedtBcNT5gQliaXjbrGW5Mrf2ak/edit#slide=id.g759a4b2e43_0_2

### CODE REVIEW

File: app/main/forms/suppliers.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def validate(self):
 # "Add" rather than "Edit" because this information can only be set once by a supplier
 class AddCompanyRegisteredNameForm(FlaskForm):
     registered_company_name = DMStripWhitespaceStringField('Registered company name', validators=[
-        InputRequired(message="You must provide a registered company name."),
-        Length(max=255, message="You must provide a registered company name under 256 characters.")
+        InputRequired(message="Enter a registered company name."),
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change 'a' to 'your':

Enter your registered company name
Enter your registered company name under 256 characters

File: app/main/forms/suppliers.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ class AddCompanyRegistrationNumberForm(FlaskForm):
         validators=[
             Optional(),
             Regexp(r'^([0-9]{2}|[A-Za-z]{2})[0-9]{6}$',
-                   message="You must provide a valid 8 character Companies House number."
+                   message="Enter an 8 character Companies House number."
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change to:

Enter your 8 character Companies House number

File: app/main/forms/suppliers.py [hunk 3]
```diff
@@ -130,46 +130,46 @@ def validate(self):
         if not super(AddCompanyRegistrationNumberForm, self).validate():
             valid = False
         if self.has_companies_house_number.data == "Yes" and not self.companies_house_number.data:
-            self.companies_house_number.errors.append('You must enter a Companies House number.')
+            self.companies_house_number.errors.append('Enter a Companies House number.')
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change to:

Enter your Companies House number

File: app/main/forms/suppliers.py [hunk 4]
```diff
@@ -130,46 +130,46 @@ def validate(self):
...
             valid = False
 
         if self.has_companies_house_number.data == "No" and not self.other_company_registration_number.data:
-            self.other_company_registration_number.errors.append('You must provide an answer.')
+            self.other_company_registration_number.errors.append('Enter a company registration number.')
             valid = False
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change to:
Enter your company registration number

File: app/main/forms/suppliers.py [hunk 5]
```diff
@@ -130,46 +130,46 @@ def validate(self):
...
         return valid
 
 
 class CompanyPublicContactInformationForm(FlaskForm):
     company_name = DMStripWhitespaceStringField('Company name', validators=[
-        InputRequired(message="You must provide a company name."),
-        Length(max=255, message="You must provide a company name under 256 characters.")
+        InputRequired(message="Enter a company name."),
+        Length(max=255, message="Enter a company name under 256 characters.")
     ])
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change to:

Enter your company number
Enter your company name under 256 characters

File: app/main/forms/suppliers.py [hunk 6]
```diff
@@ -130,46 +130,46 @@ def validate(self):
...
+        Length(max=20, message="Enter a phone number under 20 characters.")
     ])
 
 
 class DunsNumberForm(FlaskForm):
     duns_number = DMStripWhitespaceStringField('DUNS Number', validators=[
-        InputRequired(message="You must enter a DUNS number with 9 digits."),
-        Regexp(r'^\d{9}$', message="You must enter a DUNS number with 9 digits."),
+        InputRequired(message="Enter a DUNS number with 9 digits."),
+        Regexp(r'^\d{9}$', message="Enter a DUNS number with 9 digits."),
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Enter your 9 digit DUNS number

File: app/main/forms/suppliers.py [hunk 7]
```diff
@@ -109,7 +109,7 @@ class AddCompanyRegistrationNumberForm(FlaskForm):
         default='',
         validators=[
             Optional(),
-            Length(max=255, message="You must provide a registration number under 256 characters.")
+            Length(max=255, message="Enter a registration number under 256 characters.")
```
Comment (NoraGDS) [inline app/main/forms/suppliers.py]: Change to:

Enter your 8 digit Companies House registration number

File: app/main/forms/suppliers.py (hunk 3 above)
Comment (benvand) [reply app/main/forms/suppliers.py]: (Decided not to bother as likely deleting)

File: app/main/forms/suppliers.py (hunk 4 above)
Comment (benvand) [reply app/main/forms/suppliers.py]: (Decided not to bother as likely deleting)

**Comment by benvand** [comment]
Thanks for the review @NoraGDS !

**Pull request merged**